commands/management/add_external.go

```diff
@@ -76,6 +84,15 @@ func (cmd *addExternalCommand) Run() (commands.Result, error) {
 		}
 	}
 
+	hostname, _ := os.Hostname()
+	defaultServiceName := hostname + defaultServiceNameSuffix
+
+	if cmd.Group != defaultGroupExternalExporter && cmd.ServiceName == defaultServiceName {
+		cmd.ServiceName = fmt.Sprintf("%s-%s", strings.TrimSuffix(cmd.ServiceName, defaultServiceNameSuffix), cmd.Group)
+	}
+
+	customLabels["external_group"] = cmd.Group
```

Comment:
we shouldn't add it here. We should add it in pmm-managed while generating prometheus config